### ISSUE TITLE
AoO applies to all moves

### ISSUE THREAD

**Issue by ASilver** [crawl/crawl#2478]
**PLATFORM and OS or BROWSER:**

webtiles

**DCSS VERSION AND GAME INFORMATION:**

**DESCRIPTION:**

I could be misreporting, and if so, please accept apologies. I am under the impression the new AoO is not applying only to the first move to disengage with an adjacent opponent, but ALL moves with an adjacent opponent. What this means is that there is simply no more running ever. If I am adjacent to an opponent of equal speed, there is no longer any point and I may as well make my last stand or prepare to die. While I make a step away, so do they, but they ALSO get an attack each step. Since I will be of same speed, this means I will never outrun them, and never heal as I am hit every step. Thus an adjacent opponent must either be killed or you be killed, but running is no longer a choice. I cannot say how many times I have died now with this.

**EXTRA FILES:**

**Comment by ebering** [comment]
> but ALL moves with an adjacent opponent. 

Each adjacent move gives the monster(s) you're moving away from a 1/3rd chance to make an AoO.

> If I am adjacent to an opponent of equal speed, there is no longer any point and I may as well make my last stand or prepare to die.

[There are lots of possible options in this situation](https://gist.github.com/ebering/d09a84af80b6be60db20a0a8005e5f4d), placed as a link in case you don't want spoilers and want to figure them out for yourself!

**Comment by ASilver** [comment]
These options may exist later in the game, but in d1-d8 it is rarely the case. The idea that a melee character is drowning in applicable consumables that ate IDed and useful is a myth. My Djinni Conjurer certainly had none in his venture on d:6, with a wand of flame his attempt to kill the one adjacent let 2 others catch up (the reason I had tried to flee early), and my useful IDed potions were the two Curing, but they only prolonged my life as I got hit and allowed the others to catch me and thus accelerate my demise. My god lacked piety yet to be of any use. Of course a character at XL15, well administered, should do better, but until then surviving the early stages is now much much harder. Frankly, and I have even a 15-rune win to be clear, it is to the point of not being fun, but just punishing.
